src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -248,6 +249,10 @@
 				.collect(Collectors.toList());
 	}
 
+	public static List<String> loadAllOtherTextures(Workspace workspace) {
```

Comment:
Element util is meant for mod elements, not resources. I suggest inlining this method.